pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -337,3 +375,52 @@ func gitRefInfo(repo git.Repository, dir, ref string) (buildapi.GitRefInfo, erro
 	info.Message = lines[4]
 	return info, nil
 }
+
+// WaitForBuildComplete waits for a build identified by the name to complete
+func WaitForBuildComplete(c osclient.BuildInterface, name string) error {
```

Comment:
Actually now that I noticed, `start-build` calls `BuildLogs().Get` so essentially this code runs twice both in the client and the server. I would propose tweaks to the server code instead of duplicate code running in parallel:)